client/src/document/index.scss

```diff
@@ -117,8 +117,8 @@
     }
   }
 
-  /* 
-   * top navigational elements on tutorial pages 
+  /*
+   * top navigational elements on tutorial pages
```

Comment:
Someone must have edited this file without the line-ending trim on in their IDE. Good to clean up.